core/drand_control.go

```diff
@@ -562,7 +562,52 @@ func (d *Drand) InitReshare(c context.Context, in *drand.InitResharePacket) (*dr
 // PingPong simply responds with an empty packet, proving that this drand node
 // is up and alive.
 func (d *Drand) PingPong(c context.Context, in *drand.Ping) (*drand.Pong, error) {
```

Comment:
I really like the idea of giving status -> can we put that in a clear new RPC method and call that "STATUS" with a good format such that we can insert any metadata afterwards. Ping was originally meant to be just a ping to see if host is alive, like a heartbeat.

Reply:
Done!

Reply:
Done!